### PR TITLE
MCS-351 New Options to Not Render Depth/Object Masks and Set Screen Size

### PR DESCRIPTION
**Pull request by ThomasSchellenbergNextCentury**

### CODE REVIEW

File: python_api/README.md [hunk 1]
```diff
@@ -115,10 +115,15 @@ for config_json_file_path in config_json_file_list:
...
-mcs_run_in_human_input_mode <mcs_unity_build_file> <mcs_config_json_file> <debug=False> <enable_noise=False>
+mcs_run_in_human_input_mode <mcs_unity_build_file> <mcs_config_json_file>
 ```
 
-If you want the script to save the input and output data in a new folder named after the scene, add `true` to the end of the above console command.
+Run options:
+- `--debug`
+- `--noise`
+- `--no_depth_masks`
+- `--no_object_masks`
```
[the remark this answers is not in the document]
Comment (deanwetherby) [reply python_api/README.md:125]: 2 seems like the way to go here. Render visual image only by default and if the caller wants depth or segmentation, they have to enable it.

[the remark this answers is not in the document]
Comment (ThomasSchellenbergNextCentury) [reply python_api/README.md]: Done.

File: python_api/machine_common_sense/mcs_controller_ai2thor.py [hunk 2]
```diff
@@ -784,9 +795,17 @@ def retrieve_object_output(self, object_metadata, object_id_to_color):
         )
 
     def retrieve_pose(self, scene_event) -> str:
-        # Return Agents pose from Unity in step output object
-        return scene_event.metadata['pose'] 
-        
+        pose = MCS_Pose.UNDEFINED.name
```
Comment (brianpippin) [inline python_api/machine_common_sense/mcs_controller_ai2thor.py:798]: This pose code doesn't seem to have anything to do with this ticket?  Or am I missing something?

Comment (ThomasSchellenbergNextCentury) [reply python_api/machine_common_sense/mcs_controller_ai2thor.py:798]: You're right, I forgot about this.  There was a lint error that needed fixing.  It's a little change that mimics "retrieve_return_status" below.

Comment (brianpippin) [reply python_api/machine_common_sense/mcs_controller_ai2thor.py:798]: The code looked fine, it just didn't seem to mesh with the ticket description, was why I asked.

**Pull request merged**